package.json

```diff
@@ -30,7 +31,9 @@
   "devDependencies": {
     "audio-generator": "^1.0.5",
     "audio-speaker": "^1.1.2",
-    "watchify": "^3.6.0"
+    "mic": "^2.1.1",
+    "watchify": "^3.6.0",
```

Comment:
@dfcreative 

> why budo is complex?

`watchify` is _dead_ simple. It just watches the file for changes and runs browserify on it. `budo` has a bit more of the kitchen sink, (css watching, plugin architecture, etc.)

> but budo test for me is pretty simple

It's definitely simple to run, but has a lot of stuff going on under the hood. I guess it's just personal tastes. I like minimal things (even dev tools) that have less features but are smaller and specific.

That said, if you really like `budo` I don't want to force anyone into my preferences. :) I'm happy to update this to budo if you really want it. 👍